lib/karafka/process.rb

```diff
@@ -1,13 +1,15 @@
+# frozen_string_literal: true
+
 module Karafka
   # Class used to catch signals from ruby Signal class in order to manage Karafka shutdown
   # @note There might be only one process - this class is a singleton
   class Process
     include Singleton
 
     # Signal types that we handle
-    HANDLED_SIGNALS = %i(
+    HANDLED_SIGNALS = %i[
```

Comment:
The question is, how should it restart? Please create a ticket for that and let's roll it later as it also requires updating karafka-capistrano hooks.